### DISCUSSION TITLE
Add Adobe Illustrator

### DISCUSSION THREAD

**Discussion by ssergio-ll** [LinSoftWin/Photoshop-CC2022-Linux#58] [Ideas]
Hello.

Can you also add support for Adobe Illustrator? Does it work?

Thanks.

**Comment by MiMillieuh** [comment]
> Hello.
> 
> Can you also add support for Adobe Illustrator? Does it work?
> 
> Thanks.

https://youtu.be/jrcjCZZmD5Y

I already make it working. But I have to package it and make the installer and also this time secure my code.
